### PR TITLE
[RFR] Transaction queries

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -314,6 +314,50 @@ Creates a query to select an entity with selectorField IN values and keep the OR
...
 
+### transaction helper
+
+Simple helper to manage transaction
+You must retrieve a client with `pool.connect()` to use those.
+
+#### begin
+
+```js
+begin()();
```
Comment (djhi) [inline README.md]: Where does `Begin` come from ? You should show an import example too. Same for the others

**Comment by ThieryMichel** [comment]
For the full example, it's planned.

File: README.md [hunk 2]
```diff
@@ -322,6 +322,54 @@ Creates a query to select an entity with selectorField IN values and keep the OR
...
+### transaction helper
+
+Simple helper to manage transaction
+You must retrieve a client with `pool.connect()` to use those.
+
+#### begin
+
+```js
+import begin from 'co-postgres-queries/queries/begin';
+begin()();
```
Comment (djhi) [inline README.md]: Why the double `()` ?

File: README.md [hunk 3]
```diff
@@ -322,6 +322,54 @@ Creates a query to select an entity with selectorField IN values and keep the OR
...
+// { sql: 'BEGIN' }
+```
+
+create a query to start a transaction
+
+#### commit
+
+```js
+import commit from 'co-postgres-queries/queries/commit';
+commit()();
```
Comment (djhi) [inline README.md]: same

**Pull request merged**